app/serializers/es_form_serializer.rb

```diff
@@ -0,0 +1,62 @@
+include ActiveModel::Serialization
+
+class ESFormSerializer < ActiveModel::Serializer
+  attribute :id
+  attribute :name
+  attribute :version_independent_id
+  attribute :version
+  attribute :status
+  attribute :category
+  attribute :description
+  attribute :updated_at, key: :updatedAt
+  attribute :created_at, key: :createdAt
+  attribute :suggest
+  attribute :updated_by, key: :updatedBy
+  attribute :created_by, key: :createdBy
+  attribute :questions
+  attribute(:codes) { codes }
+
+  def codes
+    [] # object.concepts.collect { |c| CodeSerializer.new(c).as_json }
```

Comment:
It appears that codes will always be an empty array. Did you mean to leave the code after that commented?